evap/rewards/templates/rewards_reward_point_redemption_events.html

```diff
@@ -13,6 +13,7 @@
         </div>
         <div class="col text-end">
             {% translate 'Available reward points' %}: {{ total_points_available }}
+            <a href="{% url 'rewards:reward_points_export' %}" class="btn btn-dark btn-sm ms-2">{% translate 'Export reward points' %}</a>
```

Comment:
```suggestion
            <a href="{% url 'rewards:reward_points_export' %}" class="btn btn-light btn-sm ms-2">{% translate 'Export reward points' %}</a>
```

Please use a `btn-light`. `btn-dark` is currently used for actions that create/add something.